src/SdpGenerator.c

```diff
@@ -301,6 +301,14 @@ static PSDP_OPTION getAttributesList(char*urlSafeAddr) {
 
         snprintf(payloadStr, sizeof(payloadStr), "%u", EncryptionFeaturesEnabled);
         err |= addAttributeString(&optionHead, "x-ss-general.encryptionEnabled", payloadStr);
+
+        // Enable YUV444 if requested
+        if (NegotiatedVideoFormat & SCM_MASK_YUV444) {
```

Comment:
This should be `VIDEO_FORMAT_MASK_YUV444` rather than `SCM_MASK_YUV444`

Reply:
Fixed, silly mistake.
Also, just to confirm. Is it alright to put this under `if (IS_SUNSHINE())` check?
I'm not sure how Wolf is detected, and if I remember correctly it was having problems with unrecognized SDP attributes some time ago.

Reply:
Yeah, `IS_SUNSHINE()` should be fine for Wolf. I think they took care of the strict SDP checks but @ABeltramo can answer for sure.

Reply:
Thanks for thinking about Wolf too!  
Yeah the grammar is not as strict as it once was, there should be no problem in parsing any extra/unexpected options.